projects/storefrontstyles/scss/components/form-components/_item-counter.scss

```diff
@@ -39,6 +39,11 @@
       color: var(--cx-color, var(--cx-g-color-primary));
     }
 
+    &:focus {
+      color: var(--cx-color, var(--cx-g-color-primary));
+      cursor: pointer;
```

Comment:
I would also increase font-weight when focused. Changing color only is not enough.